apps/segment/segment.js

```diff
@@ -584,6 +588,18 @@ function uploadModel() {
         var modelInput = tf.io.browserFiles([topology.files[0], ...weights.files]);
       }
 
+      //Check if model with same name is previously defined
```

Comment:
should fix the style build
```suggestion
      // Check if model with same name is previously defined
```